packages/truffle-core/lib/commands/watch.js

```diff
@@ -22,6 +23,18 @@ const command = {
         /[\/\\]\./ // Ignore files prefixed with .
       ]
     };
+    // Certain large codebases have trouble with the watch command.
+    // Installing watchman resolves some of these issues.
+    if (commandExistsSync("watchman")) {
+      watchOptions.watchman = true;
+    } else {
+      config.logger.log(
+        colors.yellow(
+          "If you have trouble using watch, try installing watchman."
+        )
```

Comment:
If only displaying this message in a `catch` is not an option, can we remove the `yellow` here? That way the message will be more ignorable, but still easy enough to find if you're looking at an unhandled exception.

Reply:
Sounds fine to me